interface/patient_file/summary/disclosure_full.php

```diff
@@ -61,111 +61,115 @@
+                <?php echo xlt('for'); ?>&nbsp;
+                <a href="../summary/demographics.php" onclick="top.restoreSession()">
+                    <?php $pname = getPatientName($pid);
+                    echo text($pname); ?>
+                </a>
+            </h2>
+        </div>
+        <div class="col-12">
+            <a href="record_disclosure.php" class="btn btn-primary iframe" onclick="top.restoreSession()"><?php echo xlt('Record'); ?></a>
+            <a href="demographics.php" class="btn btn-primary" onclick="top.restoreSession()"> <?php echo xlt('View Patient') ?></a>
+        </div>
+        <div class="col-12 jumbotron mt-3 p-4">
+            <?php
+            $N = 15;
+            $offset = $_REQUEST['offset'];
+            if (!isset($offset)) {
+                $offset = 0;
+            }
+
+            $disclQry = " SELECT el.id, el.event, el.recipient, el.description, el.date, CONCAT(u.fname, ' ', u.lname) as user_fullname FROM extended_log el" .
+            " LEFT JOIN users u ON u.username = el.user " .
+            " WHERE el.patient_id = ? AND el.event IN (SELECT option_id FROM list_options WHERE list_id='disclosure_type' AND activity = 1)" .
+            " ORDER BY el.date DESC ";
+            $r2 = sqlStatement($disclQry, array($pid));
+            $totalRecords = sqlNumRows($r2);
+
+            $disclInnerQry = " SELECT el.id, el.event, el.recipient, el.description, el.date, CONCAT(u.fname, ' ', u.lname) as user_fullname FROM extended_log el" .
+            " LEFT JOIN users u ON u.username = el.user" .
+            " WHERE patient_id = ? AND event IN (SELECT option_id FROM list_options WHERE list_id = 'disclosure_type' AND activity = 1)" .
+            " ORDER BY date DESC LIMIT " . escape_limit($offset) . " , " . escape_limit($N);
+
+            $r1 = sqlStatement($disclInnerQry, array($pid));
+            $n = sqlNumRows($r1);
+            $noOfRecordsLeft = ($totalRecords - $offset);
+            if ($n > 0) {?>
+                <table class="table table-borderless">
+                    <tr>
+                        <td colspan='5'><a href="disclosure_full.php" id='Submit' onclick="top.restoreSession()"><span><?php echo xlt('Refresh'); ?></span></a></td>
+                    </tr>
+                </table>
+            <div id='pnotes'>
+                <table class="table table-borderless">
+                    <tr class="showborder_head">
+                        <th style='width: 120px';>&nbsp;</th>
+                        <th style="border-style: 1px solid var(--black)" width="140px"><?php echo xlt('Recipient Name'); ?></th>
+                        <th style="border-style: 1px solid var(--black)" width="140px"><?php echo xlt('Disclosure Type'); ?></th>
+                        <th style="border-style: 1px solid var(--black)"><?php echo xlt('Description'); ?></th>
+                        <th style="border-style: 1px solid var(--black)"><?php echo xlt('Provider'); ?></th>
```

Comment:
Hi @tywrenn 

Is the `border-style:` can be changed to the bootstrap built class?

Reply:
Yeah it can be changed to that. tbh I didn't even know CSS had a border-style declaration xD